Windows bug: Wraith incorrectly determines absolute path to before_capture file

Hi when i add before_capture: 'javascript/global.js' in config.

I get following error 
Error: Cannot find module 'C:/Code/Wraith/C:/Code/Wraith/javascript/global.js'

![wraith](https://cloud.githubusercontent.com/assets/12443892/12139216/d08fcee2-b484-11e5-972c-274ec5c4d177.png)

what am i doing wrong.could you please help.

So, i am not the only one chasing this issue. i have been breaking my head to figure out thi s one for the past 3 days. i get "Error: Cannot find module 'C:/wraith/C:/wraith/javascript/disable_javascript--phantom.js'"

--- a/lib/wraith/helpers/utilities.rb
+++ b/lib/wraith/helpers/utilities.rb
@@ -6,6 +6,9 @@ def convert_to_absolute(filepath)
   elsif filepath[0] == "/"
     # filepath is already absolute. return unchanged
     filepath
+  elsif filepath.match(/^[A-Z]:\/(.+)$/)
+    # filepath is an absolute Windows path, e.g. C:/Code/Wraith/javascript/global.js. return unchanged
+    filepath
   else
     # filepath is relative. it must be converted to absolute
     "#{Dir.pwd}/#{filepath}"

--- a/spec/helper_spec.rb
+++ b/spec/helper_spec.rb
@@ -1,0 +1,31 @@
+require "_helpers"
+
+describe "Wraith helpers classes and functions" do
+
+  describe "the convert_to_absolute function" do
+    it "should return false if no filepath provided" do
+      expect(convert_to_absolute(nil)).to eq 'false'
+    end
+
+    it "should convert a relative path to absolute" do
+      relative = 'my/filepath'
+      absolute = convert_to_absolute relative
+      expect(absolute[0]).to eq '/'
+      expect(absolute.length).to be > relative.length
+      expect(absolute).to match(/\/(.+)\/(.+)\/my\/filepath/)
+    end
+
+    it "should leave an absolute path unchanged" do
+      relative = '/my/filepath'
+      absolute = convert_to_absolute relative
+      expect(absolute).to eq relative
+    end
+
+    it "should leave a Windows-flavoured absolute path unchanged" do
+      relative = 'C:/Code/Wraith/javascript/global.js'
+      absolute = convert_to_absolute relative
+      expect(absolute).to eq relative
+    end
+  end
+
+end

